zookeeper/zookeeper.service

```diff
@@ -3,8 +3,8 @@ Description=Zookeeper
 After=network.target
 
 [Service]
-User=treadmld
-Group=treadmld
+User=zookeeper
+Group=zookeeper
 EnvironmentFile=/etc/sysconfig/zookeeper
 ExecStart=/bin/java -cp $CLASSPATH $LOG4J $JVMFLAGS $ZOOMAIN $ZOOCFG
```

Comment:
Add PreExecStart to do `"echo $IDX > /var/lib/zookeeper/myid"`

Reply:
Added!